package.json

```diff
@@ -52,11 +52,11 @@
     "npm": "^8.3.0",
     "qr-code-styling": "^1.6.0-rc.0",
     "randomcolor": "^0.6.2",
-    "react": "^17.0.1",
+    "react": "17.0.1",
```

Comment:
the small change regarding react and react-dom version was necessary to fix peer dependency issues. The upgrade to newer npm dependencies will be resolved in a different PR.